temporal-sdk/src/main/java/io/temporal/client/WorkflowClient.java

```diff
@@ -568,6 +568,288 @@ static <A1, A2, A3, A4, A5, A6, R> WorkflowExecution start(
     return WorkflowClientInternalImpl.start(workflow, arg1, arg2, arg3, arg4, arg5, arg6);
   }
 
+  /**
+   * Executes zero argument workflow with void return type together with an update workflow request.
+   *
+   * @param workflow The only supported value is method reference to a proxy created through {@link
+   *     #newWorkflowStub(Class, WorkflowOptions)}.
+   * @param updateOperation update workflow operation
+   * @return WorkflowUpdateHandle that can be used to get the result of the update
+   */
+  static <R> WorkflowUpdateHandle<R> updateWithStart(
```

Comment:
Not the biggest fan of only this SDK only having an explicit, single-purpose update-with-start call, but I guess if it's limited to just Java and all other SDKs can do better, at least the blast radius around this decision is limited.

Reply:
@cretz We should still consider this approach for other SDKs.  But we don't have to decide now.
Would be curious to discuss hear what you think are the concrete user downsides of this.  
If it's just a question of lowering method count, I'm not super concerned.

Reply:
Specific to this method and not past discussions about multi-op, the primary downside is that we are expanding the number of ways to start a workflow. I think there is developer clarity value in having common, documented ways to start a workflow. For instance, I am glad we created a workflow ID conflict policy instead of being tempted to add a `getOrStart` call. Hopefully we can limit this to signal with start and update with start and there is not a `startPaused` or similar in the future.

Since we can't have a common start call, it just made more sense to at least have a common start-with-operation call.